language/go/testdata/platforms/BUILD.want

```diff
@@ -57,6 +57,5 @@ go_test(
         "generic_test.go",
         "suffix_linux_test.go",
     ],
-    _gazelle_imports = [],
-    embed = [":platforms"],
+    _gazelle_imports = ["example.com/repo/platforms"],
```

Comment:
Could you also add a test in `cmd/gazelle/integration_test.go`? It should confirm that if there are only external test sources, the `embed` attribute is removed from an existing `go_test`, and the same label is added to `deps`, assuming the external test actually imports the library under test.

Also please manually confirm that builds. IIRC, everything in `embed` is effectively also in `deps` for the external test archive, so if `embed` is removed, that will cause a missing import error unless the same dependency is also added to `deps`.

Reply:
Manually confirmed, with both the test and library imports a package, Gazelle add the package to the deps of both `go_test` and `go_library`, even though the former embeds the latter.